aip/general/0133.md

```diff
@@ -194,17 +175,32 @@ NOT_FOUND errors][permission-denied].
 - For ensuring idempotency in `Create` methods, see [AIP-155][].
 - For naming resources involving Unicode, see [AIP-210][].
 
+## Rationale
+
+### Requiring user-settable ids
+
+[IaC][] clients use the resource ID as a way to identify a resource for applying
+updates and for conflict resolution. The lack of a user-settable ID means a
+client is unable to find the resource unless they store the identifier locally,
+and can result in re-creating the resource.
```

Comment:
There's also the issue of the id changing in the case that the resource is deleted and re-created, which could also require changing all resources that refer to it.